rest-api-spec/src/main/resources/rest-api-spec/api/indices.prepare_clone.json

```diff
@@ -0,0 +1,35 @@
+{
+  "indices.prepare_clone": {
```

Comment:
Naming is hard, but important. I don't this should be called `prepare_clone` because that would logically link it to the clone operation which is _not_ what this does since clone hard-links segments whenever possible. I think something along the lines of `prepare_reindex_target` would be more accurate.

Reply:
Sure. Will update it.